Planomatic/DeliverablesPage.xaml.cs

```diff
@@ -81,7 +81,27 @@ private void OpenItem_Click(object sender, RoutedEventArgs e)
             }
             else
             {
-               
+
+            }
+        }
+
+        private void TeamsComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            ComboBox teamsComboBox = sender as ComboBox;
+            Deliverable selectedDeliverableItem = this.AllDeliverableList.CurrentItem as Deliverable;
+            string prevTeam = selectedDeliverableItem.Team;
+            string newTeam = teamsComboBox.SelectedItem as string;
+
+            if (!string.Equals(prevTeam, newTeam, StringComparison.OrdinalIgnoreCase))
+            {
+                myApp().UpdateStatus($"Team Area path changed, was: {prevTeam}, now: {newTeam}");
+
+                selectedDeliverableItem.AreaPath = selectedDeliverableItem.AreaPath.Replace(

```

Comment:
If the area path had a sub path under it (as some teams do use), then the update would fail because the new area path with the substituted team wouldn't exist.  Verified this is the case, and that we give a nice status message that the item has failed.

Reply:
Fixed. Also i missed a scenario 
there could be 3 scenarios:
1. item path is under one of the paths in the config file
2. item path is under a sub node of one of the paths in the config file (this is the bug i fixed)
3. item path is under a node that is not part of the paths in the config file (i.e. <unmapped>) 
for the 3rd case, we first need to add the 'unmapped' option to the combobbox (to be able to show it in the UI) and ignore if user select it. then if the user wants to map it to a different node, we allow it. Note: we assume that all nodes are under myConfig().RootNode.